### PR TITLE
1.11.5 RN

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-41069

**NOTE:**  @jenafermiller will provide a link to the customer advisory notice with a proposed workaround. This link will be added to 1.11.5 RN on Friday (Aug 31, 2018).
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.11/release-notes/1.11.5/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+DC/OS 1.11.5 was released on September 4, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Issues Fixed in DC/OS 1.11.5
+
```
Comment (yishan-lin) [inline pages/1.11/release-notes/1.11.5/index.md]: Gentle reminder to add in a section here to prepare for the link to the Service Cloud announcement featuring the Customer Advisory that will come from Razi (Networking PM)/Jena (Support)/me.

Comment (yishan-lin) [reply pages/1.11/release-notes/1.11.5/index.md]: ##Customer Advisory
[loren ipsum]

##Frontend
etc...

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: Done.

**Comment by anithamuthu79** [comment]
@orsenthil I request your review as a second pair of eyes to verify the change log links.

File: pages/1.11/release-notes/1.11.5/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+- DCOS-38225 - Prevent task loss by fixing error handling failures of Mesos-IAM interaction. [enterprise type="inline" size="small" /]
+- DCOS-40410 - Bump Mesos to nightly [1.5.x eacabd7](https://github.com/mesosphere/mesos/blob/eacabd7/CHANGELOG).
+
+## Networking
+- COPS-3279/COPS-3576/DCOS-37703/DCOS-39721 - Fix erroneous values in service addresses stats. Bump dcos-net.
+- COPS-3472/DCOS-38932 - Enable access to Marathon app via overlap network. 
+- COPS-3520/DCOS-39999 - Fix DC/OS OSS build failure that occurred due to segmentation violation.
+- DCOS-38600 - Remove deadlock when SSL sockets are simultaneously sending/receiving data.
+- DCOS-39707 - Fix clustering issues with `etcd`.
+- DCOS_OSS-3697 - Fix connectivity issue between bridged and overlay networks.
```
Comment (pavisandhu) [inline pages/1.11/release-notes/1.11.5/index.md]: Changed "bridged" to "bridge" (to be consistent with the docs).

File: pages/1.11/release-notes/1.11.5/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+- COPS-3568/DCOS-39883 - Add permissions to `dcos_diagnostics_master` to read marathon state.
+- COPS-3658/DCOS-21258 - Remove `dcos-history-service`from UI. 
+- DCOS-22194 - Fix `dcos-metrics` pkgpanda build to utilize Docker. 
+- DCOS-37454 - Fix Prometheus inconsistent output with stored metrics.
+
+[enterprise]
+## Security
+[/enterprise]
+- COPS-2988 - Provide access permissions with full capabilities to the user. 
+- COPS-3195 - Fix cluster authentication issues running in strict mode and issues arising from transient errors in the 'ip-detect' script.
```
Comment (pavisandhu) [inline pages/1.11/release-notes/1.11.5/index.md]: Format "ip-detect" as code (i.e., enclose in backticks).

File: pages/1.11/release-notes/1.11.5/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+- COPS-3505/MARATHON-8326 - Add `wipe=true` support for pod's instances endpoint to automatically drain and decomission agent nodes.
+- MARATHON-7568 - Prevent ZK credentials leak in /v2/info.
+- MARATHON-8084 - Clean up Marathon API POST header. 
+- MARATHON-8368 - Fix offer rejection statistics error when maintenance mode is enabled. 
+
+## Mesos
+- COPS-3616/DCOS-39973/DCOS-40162 - Fix container launch failures that occurred due to Mesos-bridge running out of IPs.
+- COPS-3750/DCOS-41224 - Fix for UCR container cleanup `EBUSY` issue, which restarts EdgeLB tasks.
+- DCOS-38225 - Prevent task loss by fixing error handling failures of Mesos-IAM interaction. [enterprise type="inline" size="small" /]
+- DCOS-40410 - Bump Mesos to nightly [1.5.x eacabd7](https://github.com/mesosphere/mesos/blob/eacabd7/CHANGELOG).
```
Comment (pavisandhu) [inline pages/1.11/release-notes/1.11.5/index.md]: Do you need to have "eacabd7" in the text of the link? It seems obscure. Consider removing or replacing by a meaningful phrase.

File: pages/1.11/release-notes/1.11.5/index.md (hunk 3 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: Done.

File: pages/1.11/release-notes/1.11.5/index.md (hunk 2 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: Done.

File: pages/1.11/release-notes/1.11.5/index.md (hunk 4 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: The Git tag `eacabd7` (git commit id - shortened to 7 characters) of Mesos) in `[1.5.x eacabd7]`is used to uniquely identify this version of Mesos, as we could have multiple releases with interim ‘1.5.x’ Mesos releases (each with different tags and thus content). So, I don't think it is necessary to make this change.

File: pages/1.11/release-notes/1.11.5/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+- DCOS-39260 - Fix LDAP login failure after upgrade from 1.11.1 to 1.11.3.
+- DCOS_OSS-3932 - Update Java version from 8u151 to 8u181.
+
+## Storage
+ - DCOS-39891 - Add a new permission to `dcos_diagnostics_master` service account user during bootstrap.
+
+
+# Notable Changes in DC/OS 1.11.5
+
+- Update DC/OS UI for [1.11+v1.19.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.19.0/CHANGELOG.md). [oss type="inline" size="small" /]
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.5/index.md]: DC/OS UI changes need not be mentioned in notable changes, Anita. (I think it is a mistake that it is mentioned in CHANGELOG.md under Notable changes). You can include them in bug fixes.
And move the Notable changes upto the top of this page.

File: pages/1.11/release-notes/1.11.5/index.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+- Update DC/OS UI for [1.11+v1.19.0+1c67f4b5](https://github.com/mesosphere/dcos-ui-plugins-private/compare/1.11+v1.15.0+3231764b...1.11+v1.19.0+1c67f4b5). [enterprise type="inline" size="small" /]
+
+DC/OS 1.11.5 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/mesosphere/mesos/blob/eacabd7/CHANGELOG).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.5/index.md]: The link of the Mesos change log is this
https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG 

Notice how this corresponds to the ref here - https://github.com/dcos/dcos/blob/1.11.5/packages/mesos/buildinfo.json#L11

File: pages/1.11/release-notes/1.11.5/index.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+- Marathon 1.6.544 [change log](https://github.com/mesosphere/marathon/releases/tag/v1.6.544).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.5/index.md]: The link to Marathon Changelog is broken, Lets request @kensipe to tag a release for the release notes.

File: pages/1.11/release-notes/1.11.5/index.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+- Metronome 0.4.3 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.3).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.5/index.md]: +1. This is good.

File: pages/1.11/release-notes/1.11.5/index.md (hunk 5 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: @orsenthil `Notable Changes` section should be placed below `Fixed Issues` section, as per the layout we've discussed in our RN process improvement meeting. Let mw know, if you still think this move is essential.

File: pages/1.11/release-notes/1.11.5/index.md (hunk 7 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: @kensipe ^^

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: @kensipe ^^

**Comment by anithamuthu79** [comment]
@orsenthil  I've addressed your comments. One item is pending: Marathon change log link from @kensipe.

File: pages/1.11/release-notes/1.11.5/index.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+excerpt: Release notes for DC/OS 1.11.5
+---
+
+DC/OS 1.11.5 was released on September 12, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Notable Changes in DC/OS 1.11.5
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.5/index.md]: This is better, @anithamuthu79  - This is consistent with what we had for 1.11.4

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.5/index.md]: Sure.

**Pull request merged**